be/src/vec/exec/format/parquet/schema_desc.cpp

```diff
@@ -136,6 +136,8 @@ Status FieldDescriptor::parse_node_field(const std::vector<tparquet::SchemaEleme
         transform(t_schema.name.begin(), t_schema.name.end(), lower_case_name.begin(), ::tolower);
         node_field->name = lower_case_name;
         node_field->type.type = TYPE_ARRAY;
+        node_field->type.children.push_back(child->type);
```

Comment:
we already add functions in pr: https://github.com/apache/doris/pull/16978, please use it.

Reply:
done, thanks